README.md

```diff
@@ -1,4 +1,11 @@
-# `agora`
+<h1 align="center">Agora</h1>
+<br />
+<div align="center">
```

Comment:
This `align="center"` doesn't seem to work. At least in the preview on github.

Reply:
Try the [full-file preview](https://github.com/agora-org/agora/blob/9f7027a7902ef3de234624b55add9d9c000f6f83/README.md), sometimes the snippet preview is messed up.